syntax = "proto3";



message HeartbeatRequest {
  string service_id = 1;
  string timestamp = 2;
}

message HeartbeatResponse {
  string status = 1; 
}

service HeartbeatService {
  rpc EnviarHeartbeat(HeartbeatRequest) returns (HeartbeatResponse);
}
